src/tic_tac_toe/board.clj

```diff
@@ -2,10 +2,16 @@
 
 (def space " ")
 
-(defn init []
-  vector (replicate 9 space))
+(defn get-size [width]
+	(* width width))
+
+(defn get-new-board [width]
+  (repeat (get-size width) space))
 
 (defn place-marker [board position marker]
   (if (= space (get board position))
     (assoc board position marker)
     nil))
+
+(defn full? [board]
+	(not-any? #(= space %) board))
```

Comment:
Small nitpick: It seems like some of these have 2 spaces and some have more (4 or 6?) This might bug some people with OCD 😄